test/behaviour/graql/GraqlSteps.java

```diff
@@ -128,238 +136,226 @@ public void graql_delete_throws_exception(String exception, String deleteQuerySt
-//        }
+    @When("get answers of graql insert")
+    public void get_answers_of_graql_insert(String graqlQueryStatements) {
+        final GraqlInsert graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements));
+        clearAnswers();
+        answers = tx().query().insert(graqlQuery).collect(Collectors.toList());
+    }
+
+    @When("get answers of graql match")
+    public void graql_match(String graqlQueryStatements) {
+        final GraqlMatch graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements)).asMatch();
+        clearAnswers();
+        answers = tx().query().match(graqlQuery).collect(Collectors.toList());
     }
 
     @When("graql match; throws exception")
     public void graql_match_throws_exception(String graqlQueryStatements) {
-        // TODO
+        assertThrows(() -> graql_match(graqlQueryStatements));
+    }
+
+    @When("get answer of graql match aggregate")
+    public void graql_match_aggregate(String graqlQueryStatements) {
+        final GraqlMatch.Aggregate graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements)).asMatchAggregate();
+        clearAnswers();
+        numericAnswer = tx().query().match(graqlQuery).get();
+    }
+
+    @When("get answers of graql match group")
+    public void graql_match_group(String graqlQueryStatements) {
+        final GraqlMatch.Group graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements)).asMatchGroup();
+        clearAnswers();
+        answerGroups = tx().query().match(graqlQuery).collect(Collectors.toList());
+    }
+
+    @When("get answers of graql match group aggregate")
+    public void graql_match_group_aggregate(String graqlQueryStatements) {
+        final GraqlMatch.Group.Aggregate graqlQuery = Graql.parseQuery(String.join("\n", graqlQueryStatements)).asMatchGroupAggregate();
+        clearAnswers();
+        numericAnswerGroups = tx().query().match(graqlQuery).collect(Collectors.toList());
     }
 
     @Then("answer size is: {number}")
     public void answer_quantity_assertion(int expectedAnswers) {
-        // TODO
-//        assertEquals(
-//                String.format("Expected [%d] answers, but got [%d]", expectedAnswers, answers.count()),
-//                expectedAnswers, answers.count());
+        assertEquals(String.format("Expected [%d] answers, but got [%d]", expectedAnswers, answers.size()),
+                     expectedAnswers, answers.size());
     }
 
-    @Then("concept identifiers are")
-    public void concept_identifiers_are(Map<String, Map<String, String>> identifiers) {
-        for (Map.Entry<String, Map<String, String>> entry : identifiers.entrySet()) {
-            String identifier = entry.getKey();
-            String check = entry.getValue().get("check");
-            String value = entry.getValue().get("value");
+    @Then("uniquely identify answer concepts")
+    public void uniquely_identify_answer_concepts(List<Map<String, String>> answerConcepts) {
```

Comment:
This implementation, and all remaining implementations further down this file, are copied from Grakn Core.